pkg/rewrite/rewrite.go

```diff
@@ -320,6 +320,18 @@ func Rewrite(rewriteOptions RewriteOptions) error {
 		return errors.Wrap(err, "failed to write rendered")
 	}
 
+	installationBytes, err := ioutil.ReadFile(filepath.Join(u.GetUpstreamDir(writeUpstreamOptions), "userdata", "installation.yaml"))
+	if err != nil {
+		return errors.Wrap(err, "failed to read installation file")
+	}
+
+	installationFilename := kotsutil.GenUniqueKotsKindFilename(renderedKotsKinds, "installation")
+	renderedKotsKinds[installationFilename] = []byte(installationBytes)
+
+	if err := kotsutil.WriteKotsKinds(renderedKotsKinds, u.GetKotsKindsDir(writeUpstreamOptions)); err != nil {
```

Comment:
let's be consistent and move this to be done before `WriteRenderedApp` since that's how it's done in `pull`?